src/axom/quest/IntersectionShaper.hpp

```diff
@@ -276,7 +276,9 @@ class IntersectionShaper : public Shaper
   IntersectionShaper(const klee::ShapeSet& shapeSet,
                      sidre::MFEMSidreDataCollection* dc)
     : Shaper(shapeSet, dc)
-  { }
+  {
```

Comment:
Should we document the concept of the "free material" somewhere, e.g. in the class doxygen for `IntersectionShaper`, or possibly in the parent `Shaper` class ?

Reply:
I also created an issue for us to add user docs related to the shaping query -- https://github.com/LLNL/axom/issues/1053

Reply:
I added a class header that contains a bit about what IntersectionShaper does, how volume fractions are represented, and the purpose of the free material.